### ISSUE TITLE
Remove 'Most Popular' list in Services

### ISSUE THREAD

**Issue by vraravam** [ferdium/ferdium-app#16]
This term implies that Ferdium is somehow tracking all the services that have been configured by all the users - which is something that we are not doing and should not be doing anyways.

Also, from past recollection, this is just a hard-coded list of Services - nothing that is based on actual data.

Its better to remove this code

**Comment by psychowood** [comment]
My 2 cents: having a "most used" (as in "with biggest userbase, globally") top list could be a nice UX addition, especially when having lots of services available, even if it is statically defined and updated sporadically (once per year?).

There are a bunch of (non-certified) sources that could be used ([wikipedia](https://en.wikipedia.org/wiki/Instant_messaging#User_base), [general](https://www.messengerpeople.com/global-messenger-usage-statistics/) [reports](https://www.businessofapps.com/data/messaging-app-market/), app store download numbers, ...) to define the list.

**Comment by Flying--Dutchman** [comment]
> There are a bunch of (non-certified) sources that could be used ([wikipedia](https://en.wikipedia.org/wiki/Instant_messaging#User_base), [general](https://www.messengerpeople.com/global-messenger-usage-statistics/) [reports](https://www.businessofapps.com/data/messaging-app-market/), app store download numbers, ...) to define the list.

Like the idea! 
In that case it should also say where the data comes from, so that users do not think that they are being tracked

**Comment by vraravam** [comment]
yes - the main idea was that we need to re-affirm to the user that we are not tracking them. However that's implemented, im fine with that.

**Comment by mydarkstar** [comment]
We could also just rename the section to be a "Recommended" list, with recipes that are known to be popular amongst the community and which work well with Ferdium.

**Comment by NathanaelGandhi** [comment]
> There are a bunch of (non-certified) sources that could be used ([wikipedia](https://en.wikipedia.org/wiki/Instant_messaging#User_base), [general](https://www.messengerpeople.com/global-messenger-usage-statistics/) [reports](https://www.businessofapps.com/data/messaging-app-market/), app store download numbers, ...) to define the list.

I like this idea from psychowood, however, an interim step could be as mydarkstar suggested, 
> We could also just rename the section to be a "Recommended" list, with recipes that are known to be popular amongst the community and which work well with Ferdium.

However '*Recommended*', just like '*Most Popular*' infers some judgment on the quality/promotion of the service. Something like "**Ferdium Picks**" would suggest that it's a list curated by the Ferdium developers (without the confusion of suggesting it is services for developers with a name like "*Dev Picks*"). To me, this appears like the lowest hanging fruit as the existing implementation is as vraravam said:
> Also, from past recollection, this is just a hard-coded list of Services - nothing that is based on actual data.
